packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -128,24 +127,15 @@ const Messages: FunctionComponent<Props> = ({
   deletingState,
   hideDeleteModal,
 }) => {
-  const [_, setThreadsPaginationOffset] = useState<
-    PaginationBody["offset"] | undefined
-  >(0)
-
   useEffect(() => {
+    console.log("messageLayoutNotifications:", messageLayoutNotifications)
```

Comment:
to delete 😉

Reply:
Fixed in [c9ebc68](https://github.com/mudita/mudita-center/pull/722/commits/c9ebc68786cb7354dcfc129f147c984f311181c5)